docs/source/backends/adios2.rst

```diff
@@ -105,6 +105,14 @@ The new layout may be activated **for experimental purposes** in two ways:
 
 The ADIOS2 backend will automatically recognize the layout that has been used by a writer when reading a dataset.
 
+.. tip::
+   This schema does not use ADIOS2 attributes any more, making the ``bpls -a`` and ``bpls -A`` switches useless for openPMD datasets.
+   Their functionality can be emulated via regexes:
+
+   * Print datasets and attributes: Default behavior
```

Comment:
Is this what we mean here?
```suggestion
   * Print datasets and attributes: Default behavior (``bpls -a -l``)
```

Reply:
The only ADIOS2 attributes that we still use are internal ones such as `__openPMD_internal/openPMD2_adios2_schema`. So, the `-a` flag has (nearly) no influence. By default behavior, I mean "what happens if I call `bpls` without further parameters).

Reply:
Ah, so
```suggestion
   * Print datasets and attributes: No extra flags needed to ``bpls``
```